atlas/portal/pkg/svc/zserver.go

```diff
@@ -0,0 +1,219 @@
+package svc
+
+import (
+	models "atlas/portal/internal"
+	"atlas/portal/pkg/pb"
+	responderpb "atlas/responder/pkg/pb"
+	"context"
+	"errors"
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"time"
+)
+
+const (
+	// version is the current version of the service
+	version            = "0.0.1"
+	invalidServiceName = "invalid service name"
+	emptyRequest       = "empty Request"
+	success            = "success"
+	errorMsg           = "error"
+	errorMissingResp   = "missing repository response error"
+	hiddenUptimeMsg    = "uptime is hidden, mode = false"
+	portal             = "portal"
+	responder          = "responder"
+	storage            = "storage"
+)
+
+type Portal struct {
+	pb.PortalServer
+	client responderpb.ResponderClient
+	logger *logrus.Logger
+	portal models.Service
+}
+
+func getPortal() models.Service {
+	return models.Service{
+		ServiceName:          "portal",
+		ServiceDesc:          "portal service desc",
+		ServiceUptime:        time.Now(),
+		ServiceCountRequests: 0,
+	}
+}
+
+func NewPortal(logger *logrus.Logger) (*Portal, error) {
+	conn, err := grpc.Dial("0.0.0.0:9091" /*+os.Getenv("PORT")*/, grpc.WithInsecure(), grpc.WithBlock())
```

Comment:
Please use config value instead of magic number

Reply:
Please keep in mind, that in kubernetes environment each service will have its own address and (possible) port, so you can't use the same values for all the services. These things should be configurable

Reply:
Done